code/__DEFINES/layers.dm

```diff
@@ -135,14 +135,14 @@
 ///AI Camera Static
 #define CAMERA_STATIC_PLANE 200
 
+///Debug Atmos Overlays
+#define ATMOS_GROUP_PLANE 450
+
 ///Popup Chat Messages
-#define RUNECHAT_PLANE 250
+#define RUNECHAT_PLANE 501
 
 /// Plane for balloon text (text that fades up)
-#define BALLOON_CHAT_PLANE 251
-
-///Debug Atmos Overlays
-#define ATMOS_GROUP_PLANE 450
+#define BALLOON_CHAT_PLANE 502
 
 ///--------------- FULLSCREEN IMAGES ------------
```

Comment:
Do you mind putting runechat and balloon above the fullscreen stuff? This sectioning is probably doing more harm then good, but I'd really like to maintain good sorting here

Reply:
ahh, alright- i think i got confused by what you meant earlier, one moment

Reply:
> above the fullscreen stuff?

and just to clarify- you mean like this, or am i misunderstanding?
![image](https://user-images.githubusercontent.com/68376391/145322043-1ce70b10-a00a-44ee-ac19-90b420c3dc89.png)

Reply:
or, i assume now, 
![image](https://user-images.githubusercontent.com/68376391/145322249-df435d40-b55a-4625-89e3-a28034ae674d.png)

Reply:
All those layers are specific to the fullscreen plane, just move rune and balloon below CURSE_LAYER, could make a new section header if you'd like too


> ahh, alright- i think i got confused by what you meant earlier, one moment

Sall good, if you misunderstood s my fault for not explaining properly

Reply:
there we go, should be all set now!